### PR TITLE
Radio Schedule - ProgramCard - Episode Title & Duration update

### LINKED ISSUE #3358: Storybook stories not showing correct Timezone/numerals
**Is your feature request related to a problem? Please describe.**
Currently when you go to Psammead Storybook and look at Radio schedules, the formating of the numerals and timezones of the dates do not match our application service configuration, so updates to the datetime formats are difficult.  https://bbc.github.io/psammead/?path=/story/components-radioschedule-rtl--rtl-default

e.g. Arabic - has inconsistent numerals:
Screenshot arabic radio schedules

**Describe the solution you'd like**
In Psammead Storybook we do not pull in our moment locale overrides. 

In our application, we do, for example for Arabic, we use our locale and timezone:
```
import '@bbc/psammead-locales/moment/ar';
import '@bbc/moment-timezone-include/tz/GMT';
```

We should update storybook examples for Radio Schedules so the correct locale and timezone are being imported for each service. 

**Describe alternatives you've considered**
N/A

**Testing notes**

Checking storybook examples should be sufficient. This PR should not change any of the code of the actual component

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by sareh**
Resolves #3358

**Overall change:** Update Radio Schedules ProgramCard - EpisodeTitle is now generated using the start timestamp, with moment, passing in the locale and timezone for that service. 

**Code changes:**

- Remove episodeTitle prop.
- Major version bump since we are removing a prop
- Generate episodeTitle via moment using start timestamp with format `LL` to match the Story Promo date format. 
- Ensure Storybook uses correct timezone for each service (from #3376)
- Ensure correct locale is used for the duration and for the alt calendar. (using version 3.0.0 of the timestamp container utilities from #3403)

You can review all of the different Radio Schedule components here in Storybook: 

**Note that any of our locale overrides (e.g. making arabic show Western Arabic numerals and dual months, pashto's entire locale, persian dual calendars) *are not going to be visible in storybook* until we do this other PR: https://github.com/bbc/psammead/issues/3358**

**afrique**
Screenshot afrique radio schedule

**arabic**
Screenshot arabic radio schedule

**hausa**
Screenshot hausa radio schedule

**korean**
Screenshot korean radio schedule

**pashto**
Screenshot pashto radio schedule

**persian**

Screenshot persian radio schedule

**somali**
Screenshot somali radio schedule

**swahili**
Screenshot swahili radio schedule

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-radio-schedule/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.0.0 | [PR#xxx](https://github.com/bbc/psammead/pull/xxx) Remove episodeTitle prop. In ProgramCard, generate episodeTitle with moment using start timeestamp. |
```
Comment (sareh) [inline packages/components/psammead-radio-schedule/CHANGELOG.md]:
```suggestion
| 2.0.0 | [PR#3377](https://github.com/bbc/psammead/pull/3377) Remove episodeTitle prop. In ProgramCard, generate episodeTitle with moment using start timeestamp. |
```

File: packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx [hunk 2]
```diff
@@ -164,13 +163,21 @@ const renderHeaderContent = ({
...
     timezone,
     locale,
-    false,
-  );
+    isRelative: false,
+  });
+
+  const episodeTitle = formatUnixTimestamp({
+    timestamp: startTime,
+    format: 'D MMMM YYYY HH:mm z',
```
Comment (tochwill) [inline packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx]: Do we want to be including the time in the episode title, as we already have it in the start time above the card?

Comment (sareh) [reply packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx]: @tochwill I've updated the format to have just the long date format `LL` that we use for timestamps on Story Promos.

File: packages/components/psammead-radio-schedule/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.0.0 | [PR#3377](https://github.com/bbc/psammead/pull/3377) Remove episodeTitle prop. In ProgramCard, generate episodeTitle with moment using start timeestamp. |
```
Comment (sareh) [inline packages/components/psammead-radio-schedule/CHANGELOG.md]:
```suggestion
| 2.0.0 | [PR#3377](https://github.com/bbc/psammead/pull/3377) Remove episodeTitle prop. In ProgramCard, generate episodeTitle with moment using startTime. |
```

**Review by tochwill — APPROVED** [review]

**Review by mmcculloch — APPROVED** [review]

**Comment by mmcculloch** [comment]
Looks good! 👍

**Review by EinsteinNjoroge — APPROVED** [review]

**Comment by sareh** [comment]
Merging in. Will do testing when it is pulled into Simorgh

**Pull request merged**